20-packages.ks

```diff
@@ -75,6 +75,13 @@ grub2-efi-x64
 grub2-efi-x64-cdboot
 shim-x64
 
+# Volume, filesystem and image management
+mdadm
+parted
+ssm
+xfsprogs
+udpcast
```

Comment:
This adds new dependency `udpcast` (currently in EPEL7) and few other useful utilities for volume management.